webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -41,15 +41,15 @@ export const getNotifications = url => dispatch => {
   );
 };
 
-export const onMarkAsRead = (group, id) => dispatch => {
+export const onMarkAsRead = (group, ids) => dispatch => {
```

Comment:
i think it makes sense to have a seperate action "onMarkGroupAsRead", that way existing action still makes sense.